setup.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+from setuptools import setup, find_packages # type: ignore
+import upf
+
+
+long_description=\
+"""============================================================
+ UPF: A library that unifies planning frameworks
+ ============================================================
+    Insert long description here
+"""
+
+setup(name='upf',
+      version=upf.__version__,
+      description='Unified planning framework',
+      author='UPF Team',
+      author_email='info@upf.com',
```

Comment:
```suggestion
      author='AIPlan4EU Organization',
      author_email='aiplan4eu@fbk.eu',
```

@mikand do you have a better suggestion?

Reply:
@Framba-Luca for now proceed with this suggestion, also in the planner wrappers